### DISCUSSION TITLE
converter.jar backward compatibility

### DISCUSSION THREAD

**Discussion by hridayahuja44** [async-profiler/async-profiler#979] [General]
Hi @apangin, 

Firstly, I’d like to understand whether the latest version of the converter is compatible with the 3.0 version of the async profiler. Specifically, are there any known issues or limitations when using the latest converter with this version of the profiler. 
Secondly, is the latest converter backward compatible with the async profiler release from January? 

This is the link to the January release for async profiler:
https://github.com/async-profiler/async-profiler/releases/tag/v3.0

Lastly, could you provide any information on whether there are plans for a new release of the async profiler? Knowing this would help us plan our usage and integration strategy accordingly.

Thanks in advance for your assistance!

**Comment by apangin** [comment]
The latest converter (from `master`) can perfectly parse recordings produced by async-profiler 3.0.

We've started preparing a new release of async-profiler, but there is no specific timeline.

**Comment by hridayahuja44** [comment]
issue is solved
